package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "manual-proj",
+    "version": "1.0.0",
+    "description": "Test manual app steps",
+    "keywords": [
+        "ui5",
+        "openui5",
+        "sapui5"
+    ],
+    "main": "webapp/index.html",
+    "dependencies": {},
+    "devDependencies": {
+        "@ui5/cli": "^2.12.0",
+        "@sap/ux-ui5-tooling": "1"
+    },
+    "scripts": {
+        "start": "fiori run --open 'index.html'",
+        "start2": "ui5 serve --config=ui5.yaml  --open index.html"

```

Comment:
Please check in the documentation whether we use fiori run or ui5 serve. Delete the other one.